includes/compat/trait-wc-stripe-subscriptions.php

```
@@ -311,6 +327,12 @@ public function process_change_subscription_payment_with_deferred_intent( $subsc
 					$payment_information['payment_method_details'],
 					$selected_payment_type
 				);
+
+				// Link the new token to the subscription so My Account renders it.
+				// Scoped to ECE here; tracked for the broader paths in #5382.
+				if ( $is_express_checkout_submission ) {
+					WC_Stripe_Express_Checkout_Helper::replace_subscription_payment_token( $subscription, $payment_method_id );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Express_Checkout_Helper' in method 'process_change_subscription_payment_with_deferred_intent'. (undefined)

(StaticAccess)